internal/devbox/devbox.go

```diff
@@ -228,15 +222,22 @@ func (d *Devbox) Shell(ctx context.Context) error {
 		WithShellStartTime(telemetry.ShellStart()),
 	}
 
-	shell, err := NewDevboxShell(d, opts...)
+	shell, err := NewDevboxShell(d, envOpts, opts...)
 	if err != nil {
 		return err
 	}
 
 	return shell.Run()
 }
 
-func (d *Devbox) RunScript(ctx context.Context, cmdName string, cmdArgs []string) error {
+// runDevboxServicesScript invokes RunScript with the envOptions set to the appropriate
+// defaults for the `devbox services` scenario.
+// TODO: move to services.go
+func (d *Devbox) runDevboxServicesScript(ctx context.Context, cmdArgs []string) error {
```

Comment:
`RunScriptWithDefaultOptions` ?

Reply:
Actually, I'm going to leave this as is, because "default options" may not be correct in all scenarios. I'll modify this in a follow up to be even more Services-specific.